cmd/container-disk-v2alpha/main.c

```diff
@@ -0,0 +1,142 @@
+#include <ctype.h>
```

Comment:
What do we need ctype for? Can we get rid of libstdc++?

Reply:
done